src/instrumenter/utils.ts

```diff
@@ -533,3 +533,12 @@ export function getTypeDesc(type: TypeNode): string {
 
     throw new Error(`Unknown type ${type.pp()} in getTypeDesc`);
 }
+
+export function isASCII(s: string): boolean {
+    for (let i = 0; i < s.length; i++) {
+        if (s.charCodeAt(i) > 127) {
+            return false;
+        }
+    }
+    return true;
+}
```

Comment:
Maybe we can use something faster and shorter, for example: https://stackoverflow.com/a/14313213/2269749?

Reply:
Fixed in [0d0da14](https://github.com/Consensys/scribble/pull/253/commits/0d0da1481ae42595676f591d33fb581891ae5846)